### DISCUSSION TITLE
EditorContent not render with Vue3

### DISCUSSION THREAD

**Discussion by Leecason** [ueberdosis/tiptap#2328] [General Questions]
Hello, I'm now developing a editor based on tiptap2 and vue3.

but now my `<EditorContent>` not render with no errors and warnings.

This code is my wrapper for tiptap content
![image](https://user-images.githubusercontent.com/17562078/147718730-ad2cdb29-73c3-4b52-962d-aff8e1d5a163.png)

but render no content
![image](https://user-images.githubusercontent.com/17562078/147718749-53161b1d-7aad-4e08-bc8f-1402e79f6b81.png)

What's the problem with it, thx!

**Comment by philippkuehn** [comment]
Hmm, looks good to me. Not sure about this. Try to provide a reproducable codesandbox.
